### DISCUSSION TITLE
[Error] Permission denied to github-actions[bot]

### DISCUSSION THREAD

**Discussion by albert-marrero** [community/community#160511] [Actions] [answered]
Hello Everyone,

The workflow I am having an issue can be found [here](https://github.com/albert-marrero/Yu-Gi-Oh-Data/actions/runs/3973700475/jobs/6812549266). The goal of the workflow is setting up Python 3.11, installing a specific Python package, running that package that produces a JSON file in a specfic directory, checking out the repo, moving the JSON file into a specific directory and pushing the code.

However, I am getting the following error:
```bash
remote: Permission to albert-marrero/Yu-Gi-Oh-Data.git denied to github-actions[bot].
```

I have checked workflow permissions under settings -> actions -> general, and the workflow have read and write permissions. I am currently at a lost with this issue and wanted to reach out to the community for help.  Also, I have attached a screenshot of the error in this discussion.

![image](https://user-images.githubusercontent.com/7494358/213851983-dabad366-7b77-4f7c-85b1-f27cc47e9020.png)

**Comment by albert-marrero** [comment, marked as answer]
The problem was I had only read-only permission set up in the workflow. Once I took it out, it worked as intended.

**Comment by chunji08** [comment]
Hi Albert, 
I am hitting the same issue as yours Today. In your description, you said "I have checked workflow permissions under settings -> actions -> general, and the workflow have read and write permissions.", but the root cause you have found  is "The problem was I had only read-only permission set up in the workflow. ", So how exactly your issue is  resolved ?  I have this set to be read & write permissions on both repo,  but the issue stays. 

Thanks, 

Jack
